### PR TITLE
Improve publishing flow

### PR DESCRIPTION
**Pull request by ClementPasteau**
Few suggestions to improve the whole publication / game management flow:

- Add a clearer "unpublish from Liluo" button on the Game in the Dashboard + better warning messages
- Add a message when publishing to indicate that the game URL stays untouched to reassure the user
- Add consolidated metrics analytics on the Game in the dashboard

Few screenshots:

Screenshot 2022-02-15 at 16 34 59

image

Screenshot 2022-02-16 at 16 03 46

Screenshot 2022-02-16 at 16 04 36

Screenshot 2022-02-15 at 16 29 03

### CODE REVIEW

File: newIDE/app/src/Export/ExportDialog/ExportHome.js [hunk 1]
```diff
@@ -66,7 +66,7 @@ const ExportHome = ({
         <div style={styles.titleContainer}>
           <Line>
             <Text size="title">
-              <Trans>Share with friends</Trans>
+              <Trans>Publish and share with friends on Liluo.io</Trans>
```
Comment (4ian) [inline newIDE/app/src/Export/ExportDialog/ExportHome.js:69]: 👍

File: newIDE/app/src/Export/GenericExporters/OnlineWebExport.js [hunk 2]
```diff
@@ -286,12 +288,22 @@ export const WebProjectLink = ({
...
+                  <RaisedButton
+                    label={<Trans>Publish this build to the game's page</Trans>}
+                    onClick={onUpdatePublicBuild}
+                  />
+                </Line>
+                <Line>
+                  <AlertMessage kind="info">
+                    <Trans>
+                      Your game URL on Liluo hasn't been updated with this
+                      build. You can update it by clicking the button above.
```
[the remark this answers is not in the document]
Comment (ClementPasteau) [reply newIDE/app/src/Export/GenericExporters/OnlineWebExport.js:302]: You're right, we're biased with technical terms. I'll review that to make it easily understandable 👍

[the remark this answers is not in the document]
Comment (4ian) [reply newIDE/app/src/Export/GenericExporters/OnlineWebExport.js:302]: > Do you think users could wonder what is "Liluo" at this point?

That's a good remark, especially when never saw before, which is the case for most/all users. We can maybe always use "Liluo.io": it's catchy and looks like a website so you can (soon) just type it on google to find it.

And write something like "[...] to the community on Liluo.io".

**Comment by Bouh** [comment]
Invert the button and the message?
It makes a jumpy button, but it also prevents spam clicks.
image

File: newIDE/app/src/GameDashboard/GameDetailsDialog.js [hunk 3]
```diff
@@ -167,6 +173,29 @@ export const GameDetailsDialog = ({
     }
   };
 
+  const unpublishGame = React.useCallback(
+    async () => {
+      if (!profile) return;
+
+      const { id } = profile;
+      try {
+        setPublicGame(null); // Public game will auto update when game is updated.
```
Comment (D8H) [inline newIDE/app/src/GameDashboard/GameDetailsDialog.js:182]: This comment can lead to confusion. I wondered it it means that a public game is always updated to Liluo.
It's public game fields that are updated in the code, right?

Comment (ClementPasteau) [reply newIDE/app/src/GameDashboard/GameDetailsDialog.js:182]: yes good point, I'll make it clearer 👍

**Review by 4ian — APPROVED** [review]

**Pull request merged**